tests/tests_app_examples/test_quick_start.py

```diff
@@ -55,15 +55,15 @@ def test_quick_start_example_cloud() -> None:
 
         def click_gradio_demo(*_, **__):
             button = view_page.locator('button:has-text("Interactive demo")')
-            button.wait_for(timeout=3 * 1000)
+            button.wait_for(timeout=60 * 1000)
```

Comment:
5 sec should be more than enough. This is the time to load the page.

Reply:
updated and it passed, I'm just worried that it can be flakey.